ValeStyles/Couchbase/An.yml

```diff
@@ -4,8 +4,7 @@ link: https://docs.couchbase.com/styleguide/articles.html
 level: error 
 
 tokens:
-  - '[Aa] \b[aAeEiIoOuU].+\b'
-  - '[Aa] \b[Hh][Tt][Mm][Ll]\b'
+  - '\b[Aa]\s+[AEIOUaeiou]\w*\b'
```

Comment:
This looks good. Word boundary, "a", any number of spaces, a word beginning with a vowel. 👍